libs/langchain-openai/src/chat_models.ts

```diff
@@ -596,9 +596,10 @@ export class ChatOpenAI<
             part.message ?? { role: "assistant" }
           ),
         };
-        if (part.finish_reason) {
-          generation.generationInfo = { finish_reason: part.finish_reason };
-        }
+        generation.generationInfo = {
+          ...(part.finish_reason ? { finishReason: part.finish_reason } : {}),
```

Comment:
Should be `finish_reason` in the `generationInfo` as well

Reply:
Fixed :+1: